README.md

```diff
@@ -121,6 +121,84 @@ The `--fix` option on the command line automatically fixes problems reported by
 | :white_check_mark: | [no-empty-attrs](./docs/rules/no-empty-attrs.md) | Prevents usage of empty attributes in ember data models |
 
 
+### Components & tests
+
+|    | Rule ID | Description |
+|:---|:--------|:------------|
+|  | [no-jquery](./docs/rules/no-jquery.md) | Disallow use of `this.# eslint-plugin-ember
```

Comment:
I know, but this is autogenerated. If we don't fix the error in the generator the next one in running the script will ruin this again.